apps/heft/src/configuration/HeftParameterManager.ts

```diff
@@ -0,0 +1,166 @@
+import { InternalError } from '@rushstack/node-core-library';
```

Comment:
License comment? As an aside we should probably put a PR gate for missing license comments if they are needed everywhere.